lib/rspec/support/method_signature_verifier.rb

```diff
@@ -85,6 +85,10 @@ def has_kw_args_in?(args)
         # contain keyword arguments?
```

Comment:
Not sure if this comment is relevant now after my changes...

Reply:
It is, but is now confusing, its saying that could args contain keyword arguments without us checking what the last argument is specifically (to allow matches), thats still true except when theres not enough arguments, then we check for the hash in the middle so to speak

Reply:
Did you want to update the comment?